docs/articles/documentation/reference/testcafe-api/browserconnection/README.md

```diff
@@ -29,7 +29,7 @@ const remoteConnection = await testcafe.createBrowserConnection();
 // Outputs remoteConnection.url so that it can be visited from the remote browser.
```

Comment:
```suggestion
// Outputs the remoteConnection.url to allow access from the remote browser.
```

Reply:
This sentence implies `any remote browser`. Can we change it to:

> Outputs the `remoteConnection.url` to allow access from a remote browser.

Reply:
Yes